test.js

```diff
@@ -5,3 +5,5 @@ console.log('Hello Universe');
 
 console.log('Hello other stuff');
 
+console.log('Hello other more stuff stuff');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/franciscodua/codacy-test/pullRequest?prid=31014)
```suggestion
console.log("Hello other more stuff stuff");
```